test/e2e/pyrolisis_test.go

```diff
@@ -91,7 +98,11 @@ func (p *PyrolisisTests) CleanupBackupStorageContainers(t *testing.T) {
 				require.NoError(t, err, "There should be no error while deleting object %s/%s", container, object)
 			}
 
-			_, err = containers.Delete(storageClient, container).Extract()
+			wait.PollImmediate(CleanupBackupContainerDeleteInterval, CleanupBackupContainerDeleteTimeout,
+				func() (bool, error) {
+					_, err = containers.Delete(storageClient, container).Extract()
```

Comment:
Instead of retrying for every error I Ouwl like it better if we only retry for 409 replies